.github/workflows/fetch_versions.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
         run: |
-          go run ./cmd/fetcher .
+          go run ./internal/cmd/fetcher .
```

Comment:
Relocated all utilities which have no real use outside this project to internal.